react/features/welcome/components/WelcomePage.native.js

```diff
@@ -153,11 +207,14 @@ class WelcomePage extends AbstractWelcomePage {
             styles.messageContainer,
             styles.hintContainer,
             {
+                // $FlowExpectedError
```

Comment:
What error does this give?

Reply:
Cannot get this.state.hintBoxAnimation because property hintBoxAnimation is missing in object literal [1].